lib/ui/screen-components/un-protected/TermsConditions/OnePara/index.tsx

```
@@ -0,0 +1,31 @@
+        <Heading heading={Para.head}/>
+        </div>
+        {Para?.paras?.map((paragraph)=>
+        { 
+           return <Paragraph paragraph={paragraph} />
+
+        })}
+        <ul className='list-disc list-inside mx-4'>
+        {
+            Para?.list?.map((item)=>
+            {
+             return <ListItem item={item}  />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)